src/components/sync-dialog.tsx

```diff
@@ -132,7 +132,7 @@ export function SyncDialog( {
 	const [ showAllFiles, setShowAllFiles ] = useState( false );
 	const [ treeState, setTreeState ] = useState< TreeNode[] >( [
 		{
-			id: 'filesAndFolders',
+			id: 'all',
```

Comment:
I think this should be `contents` instead of `all`. It seems that `all` will also restore SQLs, while `contents` will restore only the wp-content files.
Alternatively, we may need to build a more hardcoded system instead of generating the fields dynamically.